vaadin-select-flow/src/main/resources/META-INF/resources/frontend/selectConnector.js

```diff
@@ -1,13 +1,16 @@
 window.Vaadin.Flow.selectConnector = {
-initLazy: function (select) {
-    const _findListBoxElement = function() {
+initLazy: window.Vaadin.Flow.tryCatchWrapper(function (select) {
+    const component = 'Vaadin Select';
+    const repo  = 'vaadin-select-flow';
```

Comment:
Now these strings are defined twice (also at end of file). I would wrap this whole thing in a new IIFE to be able to move these constants to a better scope so you can use them in all 3 places where they are needed without duplicating the strings.

Or maybe it would be cleaner to define a helper like:
```js
const tryCatchWrapper = function(callback) {
  return window.Vaadin.Flow.tryCatchWrapper(callback, 'Vaadin Select', 'vaadin-select-flow');
};
```
Then those strings are not needed elsewhere, no need for extra constants or passing those everywhere and implementation code would be shorter and cleaner. Just use the shorter `tryCatchWrapper`. The outer IIFE is probably still needed though if we want to keep this method private.

Reply:
Good suggestion!
I was considering exposing the strings to the parent scope, but not sure if IIFE would cause some unwanted side-effects.
I've tried that approach with the latest commit over here.

Reply:
Seems to be working, updated for all PRs